javascript/sdk/lib/types.ts

```diff
@@ -399,3 +425,14 @@ export type PartnerEndpoints = Record<
     state: string;
   }
 >;
+
+export interface AuditLog {
+  event: string;
+  account_id: string;
+  user_id: string;
+  values: AuditLogValues;
+  status: ExitCode;
```

Comment:
also just wanna be clear that this status value is the http status (not a probe ExitCode status). It'll be a string, but it'll be 200, 400, 500, etc

Reply:
updated!